e2e/test/FaultInjection.cs

```diff
@@ -72,7 +72,9 @@ public static bool FaultShouldDisconnect(string faultType)
         {
             return
                 (faultType != FaultType_Throttle) &&
-                (faultType != FaultType_QuotaExceeded);
+                (faultType != FaultType_QuotaExceeded) &&
+                (faultType != FaultType_AmqpC2D) &&
```

Comment:
Why this in amqp redesign??

Reply:
This is the case when single device connection has a broken link. For single devices that would mean the fault should disconnect.

---
In reply to: [251639452](https://github.com/Azure/azure-iot-sdk-csharp/pull/770#discussion_r251639452) [](ancestors = 251639452)